guidelines/api-usage.md

```diff
@@ -0,0 +1,83 @@
+        "tiles": [
+          "https://8ib71h0627.execute-api.us-east-1.amazonaws.com/v1/{z}/{x}/{y}@1x?url=s3://covid-eo-data/bm_500m_daily/VNP46A2_V011_be_{date}_cog.tif&resampling_method=nearest&bidx=1&rescale=0%2C100&color_map=viridis"
+        ]
+      },
+      "backgroundSource": null,
+      "exclusiveWith": [
+        "agriculture",
+        "no2",
+        "co2-diff",
+        "co2",
+        "gibs-population",
+        "car-count",
+        "nightlights-hd",
+        "detection-multi",
+        "water-chlorophyll",
+        "water-spm",
+        "detections-ship",
+        "detections-plane",
+        "detections-vehicles"
+      ],
+      "swatch": {
+        "color": "#C0C0C0",
+        "name": "Grey"
+      },
+      "compare": null,
+      "legend": {
+        "type": "gradient",
+        "min": "less",
+        "max": "more",
+        "stops": [
+          "#440357",
+          "#3b508a",
+          "#208f8c",
+          "#5fc961",
+          "#fde725"
+        ]
+      },
+      "paint": null,
+      "info": "Darker colors indicate fewer night lights and less activity. Lighter colors indicate more night lights and more activity. Check out the HD dataset to see a light-corrected version of this dataset."
+    }
+```
+The dataset source tiles are under the key `source.tiles`. Items surrounded by curly braces `{` and `}` should be replaced values (eg: `/{z}/{x}/{y}` might become with  `/9/421/193`. `{date}` should be of the format `YYYYMM`, if the value of `timeUnit` is `month` , and `YYYY_MM_DD`, if the value of `timeUnit` is `day`. 
+
+Dates available for the dataset are given by the `domain` key. If the `isPeriodic` value is `True`, then `domain` will only contain 2 dates, the start and end date. Any date within that range will be valid (remember that dates can either be daily (`YYYY_MM_DD`) or monthly ( `YYYYMM`)). For example, a periodic, monthly dataset can be requested with `202001` as the `{date}` field, to get data for January 2020 and `202002` for February 2020). 
+
+The URL for requesting data from the `nightlights-viirs` data for Beijing, for xyz coordinates `z=9, x=421, y=193` on January 1st, 2020 would look something like: 
+
+[https://8ib71h0627.execute-api.us-east-1.amazonaws.com/v1/9/421/193@1x?url=s3://covid-eo-data/bm_500m_daily/VNP46A2_V011_be_2020_01_01_cog.tif&resampling_method=nearest&bidx=1&rescale=0%2C100&color_map=viridis](https://8ib71h0627.execute-api.us-east-1.amazonaws.com/v1/9/421/193@1x?url=s3://covid-eo-data/bm_500m_daily/VNP46A2_V011_be_2020_01_01_cog.tif&resampling_method=nearest&bidx=1&rescale=0%2C100&color_map=viridis)
+
+Which will display in the browser.
+
+You can even save the COG directly to your file system using the same URL: 
+```bash
+
+wget "https://8ib71h0627.execute-api.us-east-1.amazonaws.com/v1/9/421/193@1x?url=s3://covid-eo-data/bm_500m_daily/VNP46A2_V011_be_2020_01_01_cog.tif&resampling_method=nearest&bidx=1&rescale=0%2C100&color_map=viridis" -O output_filename.tiff
+```
```

Comment:
I'd omit or change this section: right now it is downloading a single tile as a TIF (not COG) which is a bit odd